support-frontend/assets/helpers/productPrice/productPrices.js

```diff
@@ -81,7 +78,12 @@ const hasDiscount = (promotion: ?Promotion): boolean %checks =>
   promotion !== undefined &&
   isNumeric(promotion.discountedPrice);
 
-function applyDiscount(price: Price, promotion: ?Promotion) {
+const hasIntroductoryPrice = (promotion: ?Promotion): boolean %checks =>
```

Comment:
What does the `%` do in this line? I haven't seen this before.

Reply:
It is to tell Flow that this is a check, some details (though not many) here: https://github.com/facebook/flow/issues/4723